File: URI-py/1043.py
# -*- coding: utf-8 -*-
a, b, c = map(float, input().split())
tri = (b-c)
per = (a+b+c)
if tri < a < (b+c):
    per = (a+b+c)
    print("Perimetro = %.1f"%per)
else:
    a = ((a+b)*c)/2
    print("Area = %.1f"%a)

File: URI-cpp/1043.cpp
#include <iostream>
#include <iomanip>
#include <math.h>

using namespace std;

int main() 
{
    float a,b,c,tri,per;
    
    cin >> a >> b >> c;

    tri = (b-c);
    per = (a+b+c);

    cout << fixed << setprecision(1);

    if (tri < a && a < (b+c))
    {
        cout << "Perimetro = " << per << endl;
    }
    else
    {
        tri = ((a+b)*c)/2.0;
        cout << "Area = " << tri << endl;
    }
    
    return 0;
}